src/index.html

```diff
@@ -1,12 +1,547 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#page-top" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="Moyo header logo"
+          class="logo__image"
+          >
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="Moyo header logo"
          class="logo__image"
        >
```
Closing tag should be under the opening